HISTORY.md

```diff
@@ -52,6 +52,7 @@
 * Added class `FragmentInfo` and functions for getting fragment information. [#1900](https://github.com/TileDB-Inc/TileDB/pull/1900)
 * Added function `Dimension::create` that allows not setting a space tile extent. [#1880](https://github.com/TileDB-Inc/TileDB/pull/1880)
 * Added APIs for getting and setting ranges of queries using a dimension name. [#1920](https://github.com/TileDB-Inc/TileDB/pull/1920)
+* Added function `Attribute::nullable` to get nullability of an attribute [#1948](https://github.com/TileDB-Inc/TileDB/pull/1948)
```

Comment:
I didn't document any of the nullable-related API changes in these sections of this file (`### C++ API` and `### C API`), mostly because there were a bunch and I didn't want to obfuscate the existing patch notes here. I'd recommend removing this line and adding the link to this PR alongside `Support for nullable attributes.` in section `## New features` since this API should have been included with that patch.

Reply:
Adjusted!